src/components/blocks/FilterOptions.tsx

```diff
@@ -102,7 +102,7 @@ export const FilterOptions = ({ filter, query, handleFilterChange, corpus_types,
       <InputRadio
         key={option}
         label={option}
-        checked={query && query[QUERY_PARAMS[filter.taxonomyKey]] && query[QUERY_PARAMS[filter.taxonomyKey]].includes(option)}
+        checked={query && query[QUERY_PARAMS[filter.taxonomyKey]] && query[QUERY_PARAMS[filter.taxonomyKey]] === option}
```

Comment:
Question: Why is `filter.taxonomyKey` no longer being treated like an array? I'm not seeing the connection with the rest of the diff. Is this a rolled in bug fix?

Reply:
Fair question, the type of the `ParsedUrlQuery` is `string | string[]`. I wrote the first solution assuming I was doing the right thing and supporting multiple filters being selected (when we ever expand the filters to be multi-select), but then I noticed a bug (see third bullet point in description) where names that were similar were being checked.

I'll move this into a separate utility that checks the type first.